src/js/utils/pathUtils.js

```diff
@@ -66,6 +66,11 @@ export const getScriptType = (path: ?Array<number>): InputScriptType => {
 export const getOutputScriptType = (path: ?Array<number>): OutputScriptType => {
     if (!Array.isArray(path) || path.length < 1) return 'PAYTOADDRESS';
 
+    // compatibility for Casa - allow an unhardened 49 path to use PAYTOP2SHWITNESS
+    if (path[0] === 49) {
+        return 'PAYTOP2SHWITNESS';
+    }
+
```

Comment:
Is this still needed? It seems to me that `getOutputScriptType` is not called anymore in Casa usecase (because of the early return in deriveOutputScript`).

Reply:
yes, it is called on params validation (before signing, before sendin it to the device)
https://github.com/trezor/connect/blob/develop/src/js/core/methods/tx/outputs.js#L39

it was the first issue, and then there was a second issue: signed tx validation :)

Reply:
so basically:
- connect checks if params are correct
- connect double-checks if signed tx is correct

Reply:
thanks!